README.md

```diff
@@ -396,6 +400,7 @@ To add a resource or contribute, please check out our contribution guide [here](
 - [Statistical concepts for machine learning](http://www-bcf.usc.edu/~gareth/ISL/)
 - [AWS Machine Learning Tools](https://www-freecodecamp-org.cdn.ampproject.org/c/s/www.freecodecamp.org/news/aws-machine-learning-tools-the-complete-guide/amp/)
 - [Introductory Primer](https://www.toptal.com/machine-learning/machine-learning-theory-an-introductory-primer)
+- [Predict Loan Eligibility using Machine Learning Models](https://towardsdatascience.com/predict-loan-eligibility-using-machine-learning-models-7a14ef904057)
```

Comment:
Although this is helpful, this is a tutorial on a very specific topic rather than a general overview of a topic so I think it's best to leave it out.

Reply:
I have omitted this resource.